dac.py

```diff
@@ -402,6 +401,89 @@ def view_users():
                            data=level_info_data)
 
 
+#  ======= JOBS =======
+@app.route('/jobs/<string:job_id>')
+def view_job(job_id):
+    session = create_session()
+    job = (session
+           .query(Job)
+           .filter(Job.job_id == job_id)
+           .one())
+
+    # PRICE CONFIG
+    price_settings = {setting['type']: setting['value']
```

Comment:
Why do we store the prices in a list in the config if we use it as a dictionary? Do we use it somewhere as a list?

Reply:
It is coming from the config file, where we store the prices as list of dicts with `type` and `job` keys. It can be changed.

Reply:
The change does require existing users to change the config, right? If so, than leave it as it is. We don't want to break compatibility for this.

Reply:
Yes, it'll break the config files.

Reply:
Updated price config format - changed to desirable format but old format still works with a deprication warning.